.github/workflows/build-deb.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+name: Build .deb
+
+on:
+  - workflow_dispatch
+  - pull_request
+
+jobs:
+  build-deb:
+    runs-on: ubuntu-latest
+    
+    container:
+      image: elementary/docker:odin-unstable
+      
+    steps:
+      - name: Install dependencies
+        run: |
+          apt update
+          apt install -y git
+
+      - name: Checkout main
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Checkout debian packaging folder
+        run: git checkout origin/deb-packaging -- debian
```

Comment:
This might be able to be replaced with another `actions/checkout@v2` call, but considering this works as is right now, no need to update it.

Reply:
Hm, this is a trick to checkout the `debian` folder from the `deb-packaging` branch into the currently checked out commit, essentially merging them. I don't think `actions/checkout@v2` has anything that can do that? Otherwise, we'd need to check out the repository again into a different folder and then copy the `debian` folder across.

Reply:
There is a way to set it to clone in a subdir and a specific branch, just not sure on the merge part. But anyway, no need to worry about it. If this works, I'm happy.